packages/next-on-pages/docs/supported.md

```diff
@@ -28,12 +26,26 @@ The [Cloudflare workers runtime supports certain Node.js APIs](https://developer
 - `util`
 - `async_hooks`
 
-### External Packages
+## External Packages
 
 You can use any external npm package in your Next.js application, but keep in mind that many packages do rely on Node.js APIs, which, as presented in the previous section could not be supported.
 
+There are also packages which although not requiring node.js APIs don't work with `@cloudflare/next-on-pages` (and/or with Cloudflare's runtime in general) for example those that dynamically evaluate javascript code ([not allowed in the Workers runtime](https://developers.cloudflare.com/workers/runtime-apis/web-standards/#javascript-standards)) or that rely on features not supported by `@cloudflare/next-on-pages`.
```

Comment:
Can we write this a bit more succinctly? Perhaps combine the first paragraph and the second into saying:

You are free to use any external npm package with your Next application as long as it doesn't require unsupported Node.js APIs (see above), or the limited number of JavaScript APIs which Cloudflare disabled due to security concerns (link to https://developers.cloudflare.com/workers/runtime-apis/web-standards/#javascript-standards).